app/controllers/ApplicationController.php

```diff
@@ -14,7 +14,7 @@ function __construct(ApplicationForm $applicationForm, SettingRepository $settin
   {
     $this->applicationForm = $applicationForm;
 
-    $this->beforeFilter('currentUser', ['only' => ['edit', 'update']]);
+    $this->beforeFilter('isOwnerOrAdmin', ['only' => ['edit', 'update', 'show']]);
```

Comment:
oh I thought we wanted them to have access this way as well, but what you said works too and then I don't think we need that filter